classwork/intro-04-classwork.qmd

```diff
@@ -51,23 +51,34 @@ augment(forested_fit, new_data = forested_train) %>%
   accuracy(truth = forested, estimate = .pred_class)
 ```
 
-All yardstick metric functions work with grouped data frames!
+Metric sets are a way to combine multiple similar metric functions together into a new function.
+
+```{r}
+forested_metrics <- metric_set(accuracy, specificity, sensitivity)
+
+augment(forested_fit, new_data = forested_train) %>%
+  forested_metrics(truth = forested, estimate = .pred_class)
+```
+
+Metrics and metric sets work with grouped data frames!
 
 ```{r}
 augment(forested_fit, new_data = forested_train) %>%
   group_by(tree_no_tree) %>%
   accuracy(truth = forested, estimate = .pred_class)
 ```
 
-Metric sets are a way to combine multiple similar metric functions together into a new function.
+## Your turn
+
+Apply the `forested_metrics` metric set to `augment()` output grouped by `tree_no_tree`.
 
 ```{r}
-forested_metrics <- metric_set(accuracy, specificity, sensitivity)
+# Your code here!
 
-augment(forested_fit, new_data = forested_train) %>%
-  forested_metrics(truth = forested, estimate = .pred_class)
 ```
 
+Do any metrics differ substantially between groups?
+
```

Comment:
I would move that to go with the other part of the instruction, before the "Your code here" chunck starts. I think it might more easily get overlooked in the current place.